extensions/wikia/ImageReview/ImageReviewEventsHooks.class.php

```diff
@@ -27,6 +51,8 @@ public static function onArticleUndelete( Title $title, $created, $comment ) {
 	public static function onArticleDeleteComplete( Page $page, User $user, $reason, $articleId ) {
 		global $wgCityId;
 
+		// TODO: should ImageReview service be notified about article delete?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)